Lombiq.HelpfulLibraries.OrchardCore/Security/SecurityOrchardCoreBuilderExtensions.cs

```diff
@@ -111,33 +112,157 @@ public static OrchardCoreBuilder ConfigureAntiForgeryAlwaysSecure(this OrchardCo
     public static OrchardCoreBuilder ConfigureSecurityDefaults(
         this OrchardCoreBuilder builder,
         bool allowInlineScript = true,
         bool allowInlineStyle = false) =>
-        builder.ConfigureSecurityDefaultsInner(allowInlineScript, allowInlineStyle, useStaticFiles: false);
+        builder.ConfigureSecurityDefaults(new ContentSecurityPolicyHeaderConfiguration
+        {
+            AllowInlineScript = allowInlineScript,
+            AllowInlineStyle = allowInlineStyle,
+        });
 
     /// <summary>
-    /// The same as <see cref="ConfigureSecurityDefaults"/>, but also registers the <see cref="StaticFileMiddleware"/>
-    /// at the end of the chain, so <c>app.UseStaticFiles()</c> should not be called when this is used. This is helpful
-    /// because <see cref="StaticFileMiddleware"/> short-circuits the call chain when delivering static files, so later
-    /// middlewares are not executed (e.g. the <c>X-Content-Type-Options: nosniff</c> header wouldn't be added).
+    /// Provides some default security configuration for Orchard Core.
     /// </summary>
+    /// <remarks>
+    /// <para>This extension method configures the application as listed below.</para>
+    /// <list type="bullet">
+    ///     <item>
+    ///         <description>
+    ///             Add <see cref="CdnContentSecurityPolicyProvider"/> to permit using script and style resources from
+    ///             some common CDNs.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Add <see cref="VueContentSecurityPolicyProvider"/> to permit script evaluation when the <c>vuejs</c>
+    ///             resource is included.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Add <see cref="ContentSecurityPolicyAttributeContentSecurityPolicyProvider"/> to amend the content
+    ///             security policy using the <see cref="ContentSecurityPolicyAttribute"/>.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Add <see cref="SkipContentSecurityPolicyProvider"/> to skip declaring a content security policy on
+    ///             responses where it makes no sense.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Add <see cref="BrowserLinkContentSecurityPolicyProvider"/> to permit accessing other ports on
+    ///             <c>localhost</c> during local development.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Make the session token's cookie always secure.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///             Make the anti-forgery token's cookie always secure.
```

Comment:
Isn't something missing from here? Others have "Add xy to ..." or "Enable the xy feature to". Or if nothing missing, then I think it should be "Make**s**". Because if you read "configures the application as listed below", it seems grammatically correct to me.

Reply:
Yeah, these are all probably better as "adds", "makes". It's not telling you to do something but lists what the method does.